Exploration shuttle redesign

MediHound opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Redesigning the Exploration Shuttle into a new shape, and tweaking some of its functionalities.

## Why It's Good For The Game

This aims to make the Shuttle a nicer place to be, and allows the people on it to move and prepare without blocking each other's path immediately.
Centralises the various functions of the ship, so medics don't need to work directly next to where cyborgs and synths recharge and pilots have to move through.

## Changelog
:cl:
tweak: Redesigning the Exploration Shuttle
/:cl:

buffyuwu commented:
ahhh we already have map PRs testmerged. when they're merged, yours will be testmerged

MediHound commented:
Ah okay, my bad! I will go ahead and update the changelog!

To keep it visible somewhere, here is the full Changelog:
tweak: Redesigning the Exploration Shuttle to give it some love, give people on it more space to work with and be able to set up themselves without having to crawl over one another. Gives medics the possibility to heal injured without being in the way of the other people.
tweak: Shuttle's hull is now symmetrical.
tweak: Medical Area opposite to the airlock.
tweak: Crew Area in the front, with seats in flight direction.
tweak: Pilot seats moved behind each other, giving access to sensor and flight consoles to one pilot, and engine console to the other pilot, without leaving an unreachable empty corner.
tweak: Cyborg Recharger moved to the rear area, second Recharger removed.
tweak: Added a sensor console in the main room, along with a seat and window to the cockpit, allowing the Pathfinder (and other explorers) to look at the sensors and thus give informed orders on where to go without people having to cramp together in the cockpit.
tweak: Removed one Anomaly Crate (from what I've been told, they don't see much use because most anomalies encountered are not able to be put into them) and one stasis cage, replaced them with two quad vehicles. (Don't think they will see much use, but it felt fitting to bring a light all-terrain wheeled vehicle on an expedition.)
bug-workaround: Added a tiny fan to the rear door as a temporary bugfix because the airlock keeps acting up. This way explorers can leave the ship at the rear door without venting a third of the ship's atmosphere.
tweak: Edited the Nebula Trader, the Debris Field, and the various planets to make sure that the landing site can accommodate the shuttle. Edited the Transition map to make sure the ship as a whole is within the boundaries for it.
